modules/PatternUtils.js

```diff
@@ -94,37 +94,29 @@ export function matchPattern(pattern, pathname) {
   }
 
   const match = pathname.match(new RegExp(`^${regexpSource}`, 'i'))
+  if (match == null) {
+    return null
```

Comment:
Are we sure we want to change the possible output signature of an exported function?